.cirrus.yml

```diff
@@ -329,10 +337,11 @@ meta_task:
         # Space-separated list of images used by this repository state
         IMGNAMES: >-
             ${FEDORA_CACHE_IMAGE_NAME}
-            ${PRIOR_FEDORA_CACHE_IMAGE_NAME}
-            ${UBUNTU_CACHE_IMAGE_NAME}
-            ${PRIOR_UBUNTU_CACHE_IMAGE_NAME}
-            ${IMAGE_BUILDER_CACHE_IMAGE_NAME}
+            # FIXME
+            #${PRIOR_FEDORA_CACHE_IMAGE_NAME}
+            #${UBUNTU_CACHE_IMAGE_NAME}
+            #${PRIOR_UBUNTU_CACHE_IMAGE_NAME}
+            #${IMAGE_BUILDER_CACHE_IMAGE_NAME}
```

Comment:
I think this is a mistake.  My understanding is that we want these VM images to NOT get pruned.  That means they need their timestamps updated by `meta_task`.  That won't happen if `meta_task` doesn't run or runs without any image names :smile: